docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+(low-priority-faqs)=
+
+# Low Priority Partition FAQs
+
+## Partition Information
+
+::::{dropdown} 1. What is the `lowpriority` partition and why is it important?
```

Comment:
Not sure on the difficulty on change the color of that. It sounds like it would be a CSS change. We can check on that. If it isn't something easily changeable, we could drop the ` ` around the lowpriority in the title. Thank you!!